pkg/region/region.go

```diff
@@ -187,3 +188,30 @@ func concatURLPaths(host string, path []string) (string, error) {
 
 	return strings.Join(elements, "/"), nil
 }
+
+//
+// Logs
+//
+
+// SetLogsBaseURL Allows overriding the Logs Base URL
+func (r *Region) SetLogsBaseURL(url string) {
+	if r != nil && url != "" {
+		r.logsBaseURL = url
+	}
+}
+
+// LogsURL returns the Full URL for the Log API (data ingest)
+func (r *Region) LogsURL(path ...string) string {
```

Comment:
Does this need an argument? I see the function being called but without any args. Futuring-facing thoughts here maybe?

Reply:
We typically have more than one endpoint per region, for Logs there's currently one per. So yes.